tests/client_test.rs

```diff
@@ -1,482 +0,0 @@
-use datago::DatagoClient;
```

Comment:
tests are moved to being directly in client.rs, seems more idiomatic to rust and actually quite practical to test in situ

Reply:
moved to another PR